src/styles/main.css

```diff
@@ -249,7 +250,17 @@
 .form-switch-checkbox:checked + .form-switch-label:before {
   @apply right-0 bg-white;
 
-  margin-right: 0.3rem;
+  margin-right: 0.05rem;
+}
+
+.form-checkbox {
+  @apply w-5 h-5 border border-solid border-gray-500 rounded-sm bg-white;
+   
+}
+
+.form-checkbox:checked {
+  @apply w-5 h-5 border-4 border-solid border-green rounded-xs bg-white;
+   
```

Comment:
The changes for `tailwind.config.js` I suggested in the comment below will work fine then we don't need `.form-checkbox` classes at all. Although, I may be wrong, please, check how it will work.

Reply:
Nice! I will check it out!